react/features/base/participants/components/styles.js

```diff
@@ -40,6 +53,7 @@ export default createStyleSheet({
     participantView: {
         alignItems: 'stretch',
         flex: 1,
-        justifyContent: 'center'
+        justifyContent: 'center',
+        overflow: 'hidden'
```

Comment:
To avoid rendering off screen, e.g. in PiP

Reply:
Then it's totally unrelated to "pinch to zoom" and needs to go into a separate commit. A description in the commit will also be nice because it's not really obvious to everyone why that's done.

Reply:
No, it's related, because it's the transformed view that gets rendered off-screen when other than default transform is applied